app/experimenter/experiments/models.py

```diff
@@ -265,10 +266,35 @@ def bugzilla_url(self):
         if self.bugzilla_id:
             return settings.BUGZILLA_DETAIL_URL.format(id=self.bugzilla_id)
 
+    def convert_date_to_milliseconds(self, date):
+        date_time = datetime.datetime.combine(
+            date, datetime.datetime.min.time()
+        )
+        date_correct_timezone = pytz.timezone("America/Los_Angeles").localize(
+            date_time
+        )
+        date_time_milliseconds = int(date_correct_timezone.timestamp()) * 1000
+
+        return date_time_milliseconds
+
     @property
     def monitoring_dashboard_url(self):
-        if self.is_begun and self.normandy_slug:
-            return settings.MONITORING_URL.format(slug=self.normandy_slug)
+        start_date = ""
+        end_date = ""
+
+        if (
+            self.status == "Live" or self.status == "Complete"
```

Comment:
Isn't this the same as `is_begun`?  Also you definitely want to avoid hardcoding strings like that, in general refer to constants like `self.STATUS_LIVE` `self.STATUS_COMPLETE`

Reply:
I changed it to `is_started` per our slack convo..